Program/FileHandling/file_logic.py

```diff
@@ -0,0 +1,14 @@
+from tkinter import filedialog
+
+class FileLogic:
+    def open_file(self, root):
+        root.filename = filedialog.askopenfilename(
+            initialdir="C:/", 
```

Comment:
Users may not necessarily have a C drive, and it's kind of unlikely they'd want this to be the initial directory anyway. 
Try `initialdir='shell:MyComputerFolder'` or read further in https://stackoverflow.com/questions/53566413/specify-this-pc-as-initial-dir-in-tkinter-open-file-popup